### PR TITLE
Add some undef checks for tags rendering

### PR DESCRIPTION
**Pull request by danny-may**
Not sure if this will fix all the issues, but hopefully will fix the current one. Cant reproduce it locally

### CODE REVIEW

File: src/backend/helpers.js [hunk 1]
```diff
@@ -60,16 +60,17 @@ const tagType = {
...
-    // text = text.replace(/([,;/])(?=\S)/g, '$1\u200b');
-    let result = converter.makeHtml(text).replace(/\n/g, '<br>');
+    if (text === undefined)
+        return undefined;
 
-    // if (result.startsWith('<p>'))
-    //     result = result.substr(3, result.length - 7);
-
-    return result;
+    const html = converter.makeHtml(text) || text;
```
Comment (HunteRoi) [inline src/backend/helpers.js:66]: the `|| text` part seems a bit security-unwise to me. Is this mdToHtml method publicly accessible in any manner?

File: src/backend/helpers.js [hunk 2]
```diff
@@ -60,16 +60,17 @@ const tagType = {
 };
 
 function mdToHtml(text) {
-    // text = text.replace(/([,;/])(?=\S)/g, '$1\u200b');
-    let result = converter.makeHtml(text).replace(/\n/g, '<br>');
+    if (text === undefined)
```
Comment (HunteRoi) [inline src/backend/helpers.js:63]: If `text` is null, we could still have issues, right?

File: src/backend/helpers.js [hunk 3]
```diff
@@ -60,16 +60,17 @@ const tagType = {
...
-    // if (result.startsWith('<p>'))
-    //     result = result.substr(3, result.length - 7);
-
-    return result;
+    const html = converter.makeHtml(text) || text;
+    return html.replace(/\n/g, '<br>');
 }
 
 function addSubtagReferences(text) {
+    if (text === undefined)
```
Comment (HunteRoi) [inline src/backend/helpers.js:71]: Same: `text` could be null

**Review by HunteRoi — CHANGES_REQUESTED** [review]

File: src/backend/helpers.js (hunk 1 above)
Comment (danny-may) [reply src/backend/helpers.js:66]: Nope, all this code is server sided.

File: src/backend/helpers.js (hunk 2 above)
Comment (danny-may) [reply src/backend/helpers.js]: Yep, but the error is specifically saying its undefined. Ive changed both checks to be nullish checks instead

File: src/backend/helpers.js (hunk 3 above)
Comment (danny-may) [reply src/backend/helpers.js]: Same as above

**Review by HunteRoi — APPROVED** [review]

**Pull request merged**